src/frontend/screens/Library/components/GamesList/index.tsx

```diff
@@ -26,62 +26,51 @@ const GamesList = ({
   onlyInstalled = false,
   isRecent = false
 }: Props): JSX.Element => {
-  const { gameUpdates, showNonAvailable } = useContext(ContextProvider)
+  const { gameUpdates } = useContext(ContextProvider)
   const { t } = useTranslation()
-  const [gameCards, setGameCards] = useState<JSX.Element[]>([])
```

Comment:
this useState and the useEffect changing it was causing extra re-renders

I moved this back to the return jsx instead of using a useState+useEffect